cmd/run_test.go

```diff
@@ -114,89 +143,3 @@ func TestRun(t *testing.T) {
 		})
 	}
 }
-
-func TestNoCleanupFlag(t *testing.T) {
-	afs := afero.NewOsFs()
-	testCases := []struct {
-		name             string
-		content          string
-		execConfig       blocks.TTPExecutionConfig
-		expectedDirExist bool
-		wantError        bool
-	}{
-		{
-			name: "Test No Cleanup Behavior - Directory Creation",
-			content: `
----
-name: test-cleanup
-steps:
-  - name: step_one
-    inline: mkdir testDir
```

Comment:
The `--no-cleanup` functionality covered by this test has been merged with the table-based test above as a new test case